src/components/buttons/CancelButton/CancelButton.tsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
-import { Button } from 'semantic-ui-react';
 import type { ButtonProps } from 'semantic-ui-react';
+import { Button } from 'semantic-ui-react';
+
+export type CancleButtonProps = ButtonProps;
```

Comment:
typo `CancleButtonProps`
```suggestion
export type CancelButtonProps = ButtonProps;
```